tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_interactive_view.ts

```diff
@@ -405,6 +406,10 @@ export class LineChartInteractiveViewComponent
     );
   }
 
+  shouldRenderTooltipPoint(point: Point | undefined): boolean {
+    return point !== undefined && !isNaN(point.x) && !isNaN(point.y);
```

Comment:
Each datum within `cursoredData: TooltipDatum[]` is typed with a `point: {x: number; y: number} | null` property.  Shouldn't this function's signature be: `shouldRenderTooltipPoint(point: Point | null): boolean` ?

Can we use `!point` instead of `point !== undefined`, to avoid thinking about the difference?

Reply:
`!point` is not good enough for type refinement and it causes type error for `point.x` (`Object is possibly 'null'.`). Changed from `undefined` to `null`.